plugins/msg-broker/mcollective/lib/openshift/mcollective_application_container_proxy.rb

```diff
@@ -34,7 +34,9 @@ class MCollectiveApplicationContainerProxy < OpenShift::ApplicationContainerProx
     # * id: string - a unique app identifier
     # * district: <type> - a classifier for app placement
     #
-    def initialize(id, district=nil)
+    def initialize(id, district=nil, blocks_multiplier=Rails.configuration.msg_broker[:quota_blocks_buffer], inodes_multiplier=Rails.configuration.msg_broker[:quota_inodes_buffer])
```

Comment:
I guess there is precedent for doing this in the `node-util/conf/watchman/plugins.d/env_plugin.rb`'s initialize method, so its up to you whether to leave it be or not.